packages/patternfly-4/react-core/src/layouts/Page/PageHeader.js

```diff
@@ -19,7 +19,9 @@ const propTypes = {
   /** True to show the nav toggle button (toggles side nav) */
   showNavToggle: PropTypes.bool,
   /** Callback function to handle the side nav toggle button */
-  onNavToggle: PropTypes.func
+  onNavToggle: PropTypes.func,
+  /** header should be condensed */
```

Comment:
Should the comment be "True if the the header is condensed"

Reply:
yes.. feel free to revise!